test/integration/taskBasedStatusUpdate.test.js

```diff
@@ -287,31 +287,53 @@ describe("Task Based Status Updates", function () {
   });
 
   describe("PATCH Integration tests for Changing the status to IDLE based on users list passed", function () {
-    let superUserId;
-    let superUserJwt;
+    let userId0;
     let userId1;
     let userId2;
     let userId3;
     let userId4;
     let userId5;
+    let userId6;
+    let userId7;
+    let userId8;
+    let userId9;
+    let superUserJwt;
     let listUsers;
     const reqBody = {};
 
     beforeEach(async function () {
-      superUserId = await addUser(userData[4]);
-      superUserJwt = authService.generateAuthToken({ userId: superUserId });
-
-      userId1 = await addUser(userData[6]);
-      userId2 = await addUser(userData[8]);
-      userId3 = await addUser(userData[9]);
-      userId4 = await addUser(userData[0]);
-      userId5 = await addUser(userData[1]);
-      listUsers = [userId1, userId2, userId3, userId4, userId5];
+      userId0 = await addUser(userData[0]);
```

Comment:
what's this id0,1.....
you could have store it in an array ?

Reply:
Implemented.